server/cluster/types.go

```diff
@@ -48,7 +48,29 @@ type CreateTableResult struct {
 }
 
 type DropTableResult struct {
-	ShardVersionUpdate ShardVersionUpdate
+	ShardVersionUpdate []ShardVersionUpdate
+}
+
+type OpenTableRequest struct {
+	SchemaName string
+	TableName  string
+	ShardID    storage.ShardID
+	NodeName   string
```

Comment:
`OpenTable` is used to open a table on a shard, `NodeName` should not be needed.

Reply:
`NodeName` is needed because we can't distinguish between leader and follower shard without it.
If `NodeName` removed form `OpenTableRequest`, it means that we can only open one table at a time on all leader and follower of the shard.